cumulusci/cli/cci.py

```diff
@@ -833,7 +836,10 @@ def org_default(config, org_name, unset):
         config.keychain.unset_default_org()
         click.echo(f"{org_name} is no longer the default org.  No default org set.")
```

Comment:
I discovered this is a funny one - you can specify *anything* for the org name, the task just unsets whatever the default org is regardless of what's entered as the org name. 🙃